src/test/objectstore/store_test.cc

```diff
@@ -2823,7 +2823,16 @@ TEST_P(StoreTest, SimpleMoveRangeDelSrcTest) {
     cerr << "move temp object" << std::endl;
     r = apply_transaction(store, &osr, std::move(t));
     ASSERT_EQ(r, 0);
-
+  }
+  if (true) {
+    r = store->umount();
+    ASSERT_EQ(0, r);
+    r = store->fsck();
```

Comment:
umount/mount perform fsck with the current settings. hence we have triple fsck here